docs/Community/sig-architecture/OWNERS

```diff
@@ -14,6 +14,7 @@ reviewers:
   - sig-release-chair
   - sig-sgx-chair
   - sig-optee-chair
+  - sig-ci
```

Comment:
Just curious, why this is not sig-ci-chair?

Reply:
It should be, thanks!